src/CodeGeneration.Roslyn.Tests/OtherFolder/NullableGeneratorTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace CodeGeneration.Roslyn.Tests.OtherFolder
+{
+    using System;
+    using System.Diagnostics;
+    using CodeGeneration.Roslyn.Tests.Generators;
+    using Xunit;
+
+    public class NullableGeneratorTests
+    {
+#nullable enable
```

Comment:
I don't know much about conventions in this regard, but I think when whole file isn't `#nullable enable`d, every explicit switch should be paired with `#nullable restore` so that in future when we migrate this project to full nullability, everything works correctly.

Also, I feel like for Facts, the directives should be inside method body.